app/proxy/SWProxy.js

```diff
@@ -165,6 +165,11 @@ class SWProxy extends EventEmitter {
   getLogEntries() {
     return this.logEntries;
   }
+
+  clearLogs() {
+    this.log({ type: 'warning', source: 'proxy', message: 'clearLogEntries ran.' })
```

Comment:
Dont forget to change that line. It doesnt really makes sense as a test, just clear the array with `this.logEntries = [];` and emit the logUpdated event in the main.js.